static/src/stylesheets/module/facia-garnett/_container.scss

```diff
@@ -453,6 +453,7 @@ $header-image-size-desktop: 100px;
     padding: 0;
     color: $brightness-46;
     text-align: right;
+    min-width: 60px;
```

Comment:
off-track question: Do we favour sass variables or specific px as a convention?

I normally use the guss-grid system as a reference, should I keep doing this?
https://github.com/guardian/guss-grid-system 🤔 😅

Reply:
As far as I know it's still good practice to reference guss when it makes sense. And as I just used 60px - it does make sense 😄 I've updated it to be gs-span(1). Thanks Kate.